generateBuildrootSBOM.py

```diff
@@ -72,48 +75,54 @@ def create_buildroot_sbom(args, br_bom: cyclonedx.model.bom):
     return br_bom_local
 
 
-def main():
+def my_main():
```

Comment:
why renamed? `main` was a good name, was it?

Reply:
That is part of implementing the pytest and following their conventions.